src/components/CommentInfo/CommentInfo.tsx

```diff
@@ -1,5 +1,29 @@
 import React from 'react';
+import { Comment } from '../../Types/comment';
 
-export const CommentInfo: React.FC = () => (
-  <>Put the comment here</>
+type Props = {
+  comment: Comment,
+};
+
+export const CommentInfo: React.FC<Props> = ({ comment }) => (
```

Comment:
Not a mistake, but a suggestion: you can always destructure props at the highest level, to avoid using .(dot) notation each time. 

```
comment: {
  name,
  email,
  body
}
```